staging/src/k8s.io/kubectl/pkg/cmd/set/set_env.go

```diff
@@ -38,15 +38,16 @@ import (
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 	"k8s.io/kubectl/pkg/polymorphichelpers"
 	"k8s.io/kubectl/pkg/scheme"
+	"k8s.io/kubectl/pkg/util/i18n"
 	"k8s.io/kubectl/pkg/util/templates"
 )
 
 var (
 	validEnvNameRegexp = regexp.MustCompile("[^a-zA-Z0-9_]")
```

Comment:
pod (po), replicationcontroller (rc), deployment (deploy), daemonset (ds), job, replicaset (rs)  is language independent. I believe this doesnt need usage of i18n.T()

Reply:
Fair point, not needed.